File: benchmarks/go/fibonacci.go
package main

import "fmt"

func fibonacci(n int) int {
    if n <= 1 {
        return n
    }
    return fibonacci(n-1) + fibonacci(n-2)
}

func main() {
    result := fibonacci(35)
    fmt.Printf("Fibonacci(35): %d\n", result)
}

File: benchmarks/c/fibonacci.c
#include <stdio.h>

int fibonacci(int n) {
    if (n <= 1) return n;
    return fibonacci(n - 1) + fibonacci(n - 2);
}

int main() {
    int result = fibonacci(35);
    printf("Fibonacci(35): %d\n", result);
    return 0;
}